pynxtools/dataconverter/readers/xps/reader.py

```diff
@@ -46,18 +46,19 @@
 
 
 CONVERT_DICT = {
-    'Instrument': 'INSTRUMENT[instrument]',
-    'Analyser': 'ELECTRONANALYSER[electronanalyser]',
-    'Beam': 'BEAM[beam]',
     'unit': '@units',
     'version': '@version',
-    'Sample': 'SAMPLE[sample]',
     'User': 'USER[user]',
-    'Data': 'DATA[data]',
+    'Instrument': 'INSTRUMENT[instrument]',
     'Source': 'SOURCE[source]',
+    'Beam': 'BEAM[beam]',
+    'Analyser': 'ELECTRONANALYSER[electronanalyser]',
     'Collectioncolumn': 'COLLECTIONCOLUMN[collectioncolumn]',
     'Energydispersion': 'ENERGYDISPERSION[energydispersion]',
-    'Detector': 'DETECTOR[detector]'
+    'Detector': 'DETECTOR[detector]',
+    'Manipulator': 'MANIPULATOR[manipulator]',
+    'Sample': 'SAMPLE[sample]',
+    'Data': 'DATA[data]',
```

Comment:
Any reason for these changes, since only only the order has been modified.

Reply:
The order was changed to reflect the order in the application definition. Probably not necessary, but may be easier to adjust once the app-def changes.